pages/index.vue

```diff
@@ -15,6 +17,17 @@ import Clock from '../components/Clock.vue'
 import Days from '../components/Days.vue'
 import Goals from '../components/Goals.vue'
 import Footer from '../components/Footer.vue'
+
+function getCookie(name) {
+  const nameEQ = name + '='
```

Comment:
What is the significance of EQ here

Reply:
Since it's not just a name anymore it has **'='** making it a **name equation**